server/infrastructure/roster_repository.py

```
@@ -0,0 +1,37 @@
+                
+                if not player_doc.exists:
+                    continue
+                
+                player_data = player_doc.to_dict()
+                seasons = player_data.get("seasons", {})
+                
+                if not seasons:
+                    continue
+                players_data[player_id] = seasons
+                
+            return players_data
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)